stores/appStore.ts

```diff
@@ -273,12 +274,16 @@ export const useAppStore = defineStore("app", {
         await this.loadScenarioResult(scenario);
       }) || []);
     },
-    async downloadExcel() {
+    async downloadExcel(comparison: boolean) {
```

Comment:
We're missing some updates to the appstore unit tests: these still say `store.downloadExcel()` whereas they should be taking an argument.